tests/tests_integration_pytest.yml

```diff
@@ -0,0 +1,82 @@
+# SPDX-License-Identifier: BSD-3-Clause
+---
+- name: Empty play to gather facts
+  hosts: all
+
+- import_playbook: playbooks/rhel7_deps.yml
+  when:
+    - ansible_distribution == "RedHat"
+    - ansible_distribution_major_version == '7'
+
+- import_playbook: playbooks/centos6_deps.yml
+  when:
+    - ansible_distribution == "CentOS"
+    - ansible_distribution_major_version == '6'
+
+- import_playbook: playbooks/centos7_deps.yml
+  when:
+    - ansible_distribution == "CentOS"
+    - ansible_distribution_major_version == '7'
+
+- import_playbook: playbooks/centos8_deps.yml
+  when:
+    - ansible_distribution == "CentOS"
+    - ansible_distribution_major_version == '8'
+
+- name: Run test playbook
+  hosts: all
+  vars:
+    - rundir: /run/system-roles-test
+  tasks:
+    - file:
+        state: directory
+        path: /run
```

Comment:
Why `/run` as opposed to a temp directory like `/tmp` or `/var/tmp`?  Or use the `tempfile` https://docs.ansible.com/ansible/latest/modules/tempfile_module.html#tempfile-module module?

Reply:
`/run` does not have any problems regarding insecure temporary files. Using tempfile has the problem that it would not be persistent so it would need to be synced all the time when re-using a machine.